src/ForeignController.sol

```diff
@@ -23,9 +22,9 @@ contract ForeignController is AccessControl {
     bytes32 public constant FREEZER = keccak256("FREEZER");
     bytes32 public constant RELAYER = keccak256("RELAYER");
 
-    bytes32 public constant LIMIT_USDC_TO_CCTP = keccak256("LIMIT_USDC_TO_CCTP");
     bytes32 public constant LIMIT_PSM_DEPOSIT  = keccak256("LIMIT_PSM_DEPOSIT");
     bytes32 public constant LIMIT_PSM_WITHDRAW = keccak256("LIMIT_PSM_WITHDRAW");
+    bytes32 public constant LIMIT_USDC_TO_CCTP = keccak256("LIMIT_USDC_TO_CCTP");
 
     IALMProxy   public immutable proxy;
     IRateLimits public immutable rateLimits;
```

Comment:
Not related to this PR, but I just realized `cctp` is not `immutable` and it should be grouped up in this group of variables probably too.

Reply:
Ah good call updated